AirCasting/AuthViews/UserSettings.swift

```diff
@@ -7,6 +7,9 @@ import UIKit
 class UserSettings: ObservableObject {
     private let userDefaults: UserDefaults
     private let crowdMapKey = Constants.UserDefaultsKeys.crowdMap
+    private let disableMappingKey = Constants.UserDefaultsKeys.disableMapping
+    private let keepScreenOnKey = Constants.UserDefaultsKeys.keepScreenOn
+    private let featureFlagsViewModel = FeatureFlagsViewModel.shared
```

Comment:
well, `UserSettings` is not a `View` so it's kinda weird it's using a `ViewModel`. We should think of a better suiting name for `FeatureFlagsViewModel`

Reply:
Hmm... Is it a Manager? 😁 Distributor?

Reply:
What is the best way to get the provider?